tests/Issues/NamespacedUse/Fixture/skip_conflict_last_name_insensitive.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Rector\Tests\Issues\NamespacedUseAutoImport\Fixture;
+
+use Some\CLASS_;
+use PhpParser\node;
+
+final class SkipConflictLastNameUsed
```

Comment:
to avoid overlap ClassReflection, better to have unique name per existing fixture:

```suggestion
final class SkipConflictLastNameUsedInsensitive
```